UI/Pages/SearchResults.cshtml

```diff
@@ -130,45 +158,77 @@
+                                            @foreach (var keyStageSubjects in 
+                                                item.SubjectsCoverage.Select(e => e.Subject).Distinct().OrderBy(e => e.KeyStageId)
+                                                    .OrderBy(e => e.Name).GroupBy(e => e.KeyStageId))
+                                            {
+                                                <li>@(((KeyStage) keyStageSubjects.Key).DisplayName()) - @keyStageSubjects.DisplayList()</li>
+                                            }
+                                        </ul>
+                                    </govuk-summary-list-row-value>
+                                </govuk-summary-list-row>
+                                <govuk-summary-list-row class="govuk-body-s">
+                                    <govuk-summary-list-row-key>
+                                        Type of tuition
+                                    </govuk-summary-list-row-key>
+                                    <govuk-summary-list-row-value data-testid="type-of-tuition">
+                                        @string.Join(", ", item.TuitionTypes.Select(e => e.Name))
+                                    </govuk-summary-list-row-value>
+                                </govuk-summary-list-row>
+                                <govuk-summary-list-row class="govuk-body-s">
+                                    <govuk-summary-list-row-key>
+                                        Tuition partner information
+                                    </govuk-summary-list-row-key>
+                                    <govuk-summary-list-row-value data-testid="results-description">
+                                        @item.Description
+                                    </govuk-summary-list-row-value>
+                                </govuk-summary-list-row>
+                                <govuk-summary-list-row class="govuk-body-s">
+                                    @{
+                                        var shortlist = Model.SelectableTuitionPartners
+                                            .First(tp => tp.SeoUrl == item.SeoUrl);
+                                    }
+                                    <govuk-summary-list-row-key>
+                                        Add to my shortlist
+                                    </govuk-summary-list-row-key>
+                                    <govuk-summary-list-row-value data-testid="results-description">
+                                        <div class="govuk-checkboxes govuk-checkboxes--small " data-module="govuk-checkboxes">
+                                            <div class="govuk-checkboxes__item shortlist-tuition-partner-checkbox">
+                                                <input class="govuk-checkboxes__input" id="@shortlist.SeoUrl" name="ShortlistedTuitionPartners"
```

Comment:
Could we prefix the `id` attribute here with `shortlist-cb` or something like that?
```suggestion
                                                <input class="govuk-checkboxes__input" id="shortlist-cb-@shortlist.SeoUrl" name="ShortlistedTuitionPartners"
```